provisions/roles/cleanup/openshift/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Remove all containers
+  sudo: yes
+  shell: docker rm -f `docker ps -a -q`
```

Comment:
we should remove all the containers other than openshift ones. but if we re resetting all the environment, we can clear out everything.